### PR TITLE
rbdl recipe

### PR DESCRIPTION
**Pull request by pariterre**

### CODE REVIEW

**Comment by pariterre** [comment]
@conda-forge/staged-recipes
I could not make it work for windows because std::isnan is not defined in early version of VS. Is there any way to compile for VS2015 at least?
Thank!

**Comment by pariterre** [comment]
@conda-forge/staged-recipes
Also no test I tried seem to work (e.g. ```test -f $PREFIX/lib/libdlib$SHLIB_EXT # [unix]```), even though everything seems fine during the compilation and the anaconda build process. Is there something wrong with this test?

File: recipes/rbdl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - eigen
+
+test:
+   commands:
+     - test 1 -eq 1  # [unix]
```
Comment (scopatz) [inline recipes/rbdl/meta.yaml]: This is not an OK test

File: recipes/rbdl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  rpaths:
+    - lib/
+    - lib/rbdl/
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - eigen
```
Comment (scopatz) [inline recipes/rbdl/meta.yaml]: Do you need to have eigen as a runtime dependency?

File: recipes/rbdl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  number: 0
+  skip: True  # [win]
+  rpaths:
+    - lib/
+    - lib/rbdl/
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
```
Comment (scopatz) [inline recipes/rbdl/meta.yaml]: You probably want make here too

Comment (scopatz) [reply recipes/rbdl/meta.yaml]: at least on unix

File: recipes/rbdl/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+   commands:
+     - test 1 -eq 1  # [unix]
+     - if not exist %LIBRARY_LIB%\\rbdl.dll exit 0  # [win]
+
+about:
+  home: https://bitbucket.org/rbdl/rbdl
+  license: zlib
+  license_file: LICENSE
```
Comment (scopatz) [inline recipes/rbdl/meta.yaml]: You should probably add `license_family: OTHER`

**Comment by scopatz** [comment]
Thanks for putting this in @pariterre

File: recipes/rbdl/meta.yaml (hunk 2 above)
Comment (pariterre) [reply recipes/rbdl/meta.yaml]: Yep, is 'host' not the right one for that?

Comment (scopatz) [reply recipes/rbdl/meta.yaml]: nope, you need another "run" section below this one :wink:

**Comment by pariterre** [comment]
@conda-forge/staged-recipes
I could not make it work for windows because std::isnan is not defined in early version of VS. Is there any way to compile for VS2015 at least?
Thank!

**Comment by pariterre** [comment]
@conda-forge/staged-recipes
Actually, it seems that it can compile for vs2015, but it does it as well with vs2008 which should be skip... How do I do that?

**Comment by pariterre** [comment]
@conda-forge/staged-recipes
appveyor doesn't work, but it is because it choose VS2008 instead of VS2015. Otherwise, it works fine :)

**Comment by scopatz** [comment]
Thanks @pariterre

**Pull request merged**